.github/workflows/rw-cdn-upload.yml

```
@@ -55,7 +55,7 @@ jobs:
 
         cd artifact
 
-        for i in $(ls); do
+        for i in $(find -type f | cut -b 3-); do
```

I think the search path is missing
```suggestion
        for i in $(find . -type f | cut -b 3-); do
```